src/content/community/language-resources/index.md

```diff
@@ -109,3 +109,9 @@ If you are bilingual and want to help us reach more people, you can also get inv
 - [Tino Group](https://wiki.tino.org/ethereum-la-gi/) - overview of Ethereum, dApps, wallets and FAQs
 - [Tap Chi Bitcoin](https://tapchibitcoin.io/tap-chi/tin-tuc-ethereum-eth) - web platform with subpages for Ethereum news and education
 - [Coin68](https://coin68.com/ethereum-tieu-diem/) - cryptocurrency portal with Ethereum news and educational content
+
+### Japanese {#ja}
+- [Japan Virtual and Crypto assets Exchange Association](https://jvcea.or.jp/)
+- [Japan Cryptoasset Business Association](https://cryptocurrency-association.org/)
+- [Crypto Investment Guide for Beginners](https://www.crypto-currencies.jp/)
+- [Wikipedia - Ethreum](https://ja.wikipedia.org/wiki/%E3%82%A4%E3%83%BC%E3%82%B5%E3%83%AA%E3%82%A2%E3%83%A0)
```

Comment:
This link is broken but not sure linking out to Wikipedia is all that useful?

```suggestion
```

Reply:
Agree here... in my opinion this is a link people already know how to get to, and could likewise be placed on every single language. Given that, I don't think it adds value listing this here.